src/index.html

```diff
@@ -5,9 +5,50 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+          crossorigin="#"
+    >
```

Comment:
Still not fixed, you can check it [HERE](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#html-formatting)
```suggestion
    <link 
      rel="preconnect"
      href="https://fonts.gstatic.com"
      crossorigin="#"
    >
```

Reply:
Fix it in all cases